build-recipe-helm

```diff
@@ -60,6 +60,19 @@ recipe_prepare_helm() {
 }
 
 recipe_build_helm() {
+    BUILD_PATH=
+    if [ -d "$BUILD_ROOT/$TOPDIR/BUILD/_service" ]; then
+      # get the packaging file name
+      FILENAME=$(cat $BUILD_ROOT/$TOPDIR/BUILD/_service | xmllint --xpath 'string(//param[@name="filename"])' -)
```

Comment:
you can not create a check that _service is a directory and afterwards reading it as an XML file.

also the "cat" command is quite pointless, you could add the file name to xmllint directly.

But what worries me most is that there should not be a source file in the BUILD directory at all, no?